manifests/config.pp

```diff
@@ -59,6 +63,15 @@
       onlyif  => "test `authselect current | grep -i profile | grep -i ${authselect_profile} | wc -l` -lt 1",
       command => "authselect select ${authselect_profile} --force",
     }
+    # FOR ITEMS IN $authselect_features ... $authselect_feature
+    $authselect_features.each | $authselect_feature | {
+      exec { "authselect_enable_feature_${authselect_feature}":
+        path    => '/sbin/:/bin/:/usr/bin/:/usr/sbin/',
+        onlyif  => "test `authselect current | grep -i -A20 features | grep -iv features | grep -i ${authselect_feature} | wc -l` -lt 1",
```

Comment:
Though it'll probably be fine the hardcoded -A20 worries me a bit. Not sure what types of values you are seeing when running `authselect current`. But you could use this:
```
authselect current | awk '/features:$/{y=1;next}y'
```
That will return all lines after a line ending with ` features:` NOT including the features line itself.

That way if the number of features gets crazy big it's not a worry, plus you can get rid of the `grep -iv features` since that won't be needed anymore.

Reply:
Thanks. That works well.